src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+Apps to be usable offline. For example, travel and airline apps should have
+trip details and boarding passes easily available when offline. Music,
+video, and podcasting apps should allow for offline playback. Social and
+news apps should cache recent content so users can read it when offline.
+Users also expect to stay authenticated when offline, so design for
+offline authentication. By allowing users to use your PWA while offline,
+you'll create a true app-like experience for them.
+
+#### How
+
+After determining which features your users expect to work offline, you'll
+need to make your content available and adaptable to offline contexts. In
+addition, you can use
+[IndexedDB](https://developers.google.com/web/ilt/pwa/working-with-indexeddb),
+an in-browser NoSQL storage system, to store and retrieve data, and
+[background sync](https://developers.google.com/web/updates/2015/12/background-sync)
+to allow users to take actions while offline and defer server communications
+until the user has a stable connection again. You can also use service
+workers to store other kinds of content, such as images, video files, and
+audio files for offline use, as well as use them to implement
+[safe, long-lived sessions](https://developers.google.com/web/updates/2016/06/2-cookie-handoff)
+to keep users authenticated. From a user experience perspective, you can use
+[skeleton screens](https://uxdesign.cc/what-you-should-know-about-skeleton-screens-a820c45a571a)
+that give users a perception of speed and content while loading that can
+then fall back to cached content or an offline indicator as needed.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Is fully accessible
+
+All user interactions pass [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
+accessibility requirements.
+
+{% endDetailsSummary %}
+
+All user interactions pass [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
+accessibility requirements.
+
+#### Why
+
+Most people at some point in their life will want to take advantage of your
+PWA in a way that is covered under the [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
+accessibility requirements. Humans' ability to interact with and understand
+your PWA exist on a spectrum and needs can be temporary or permanent. By
+making your PWA accessible, you ensure it's usable for everyone.
+
+#### How
+
+W3C's [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/)
+is a good place to start. For testing, a majority of accessibility testing
+must be done manually. Tools like the [Accessibility](/lighthouse-accessibility/)
+audits in Lighthouse, [axe](https://github.com/dequelabs/axe-core), and
+[Accessibility Insights](https://accessibilityinsights.io/) can help you
+automate some accessibility testing. It's also important to use semantically
+correct elements instead of recreating those elements on your own
+(for instance, `a` and `button` elements). This ensures that when you do need
```

Comment:
The parenthetical remark needs to go after 'those elements'. This is a grammar issue. The parenthetical needs to be the same place it would be if it were set off by commas instead of Parens.